### PR TITLE
Feature/fix labels and cukes

### PR DESCRIPTION
**Pull request by corroded**
Updates and refactors cucumbers. Added some DRYness and parameters(and defaults) to signin and signup. 

New step in cucumber:

```
When I sign up with email "test@ivolunteer.com.ph" and password "mypassword"
```

### CODE REVIEW

**Comment by corroded** [comment]
daming merge branch master. dapat `git rebase master` pala ginawa ko hindi `git merge master`

**Comment by crigor** [comment]
Ok lang merge sakin instead of rebase. Kung may kasama kang gumagawa ng branch, mukhang hindi pwede rebase. kasi parating out of sync ung branch kung ganun.

File: features/step_definitions/web_steps.rb [hunk 1]
```diff
@@ -33,12 +33,12 @@ def with_scope(locator)
...
 # Single-line step scoper
 When /^(.*) within (.*[^:])$/ do |step, parent|
-  with_scope(parent) { When step }
+  with_scope(parent) { step step }
 end
 
 # Multi-line step scoper
 When /^(.*) within (.*[^:]):$/ do |step, parent, table_or_string|
-  with_scope(parent) { When "#{step}:", table_or_string }
+  with_scope(parent) { step "#{step}:", table_or_string }
```
Comment (crigor) [inline features/step_definitions/web_steps.rb:41]: Para saan tong change na to?

File: features/login_and_registration.feature [hunk 2]
```diff
@@ -6,74 +6,20 @@ Feature: Login and registration
...
-      | City | City |
-    And I select "Philippines" from "Country"
-    And I select "Philippines" from "Nationality"
-    And I select "NCR" from "Region"
-    And I choose "Male"
-    And I check the expertise "Education"
-    And I press "Sign up"
-    Then I should be on the dashboard page
-    And I should have an expertise of "Education"
-
```
Comment (crigor) [inline features/login_and_registration.feature:76]: pinagsama mo na lang ung tests no?

**Comment by crigor** [comment]
Natry ko na to sa local. 

11 scenarios (11 passed)
72 steps (72 passed)

File: features/login_and_registration.feature (hunk 2 above)
Comment (corroded) [inline features/login_and_registration.feature:76]: yup at ginawa kong mas concise. paulit ulit lang eh hehe. yun yung brittle kasi pag me nagpalit sa registration natin sira lahat tas papalitan mo isa isa. mas ok yung isa lang papalitan mo hehe

File: features/step_definitions/web_steps.rb (hunk 1 above)
Comment (corroded) [inline features/step_definitions/web_steps.rb:41]: deprecation notice. deprecatd na yung "Given, When, Then" sa loob ng mga steps. dapat "step" na

**Comment by crigor** [comment]
Mukhang naclose lang nung una at hindi ko namerge.

**Pull request merged**